ibis/backends/spark/compiler.py

```diff
@@ -50,6 +50,12 @@ class SparkExprTranslator(ExprTranslator):
 rewrites = SparkExprTranslator.rewrites
 
 
+@rewrites(ops.FloorDivide)
```

Comment:
I like this.

But..is this related to this PR? XD

Reply:
Somehow related. This is missing in master and tests fail because the lack of registering `FloorDivide` in Spark. In master tests don't really fail, because the PySpark backend is also loaded, and looks like the PySpark backend is also registering the Spark operation. I need to change this here (or in a separate PR merged before this one), because here we load one backend at a time, and without loading PySpark, the Spark backend tests fail.

I'm already working on removing the spark backend, so this addition makes the tests pass here, but this code won't live very long, so no big deal I think, even an empty function would be ok IMO.

Reply:
kk thanks for the explanation. SGTM